src/stream/xml_reader.rs

```diff
@@ -173,6 +173,9 @@ impl<R: Read> ReaderState<R> {
                     }
                 }
                 XmlEvent::End(name) => {
+                    // https://github.com/ebarnard/rust-plist/pull/88#discussion_r1024544219
+                    // TODO: Why do we want to keep in the plist here?
+                    #[allow(clippy::wildcard_in_or_patterns)]
```

Comment:
@ebarnard you said this should stay in as a reminder. Can you explain this? I'm not quite sure what to put into the comment that should accompany this.

Reply:
Now that `quick-xml` checks end tags match start tags I think we can lose the `b"plist"` in the end.

Reply:
Done.